src/util/system.h

```diff
@@ -264,6 +264,16 @@ class ArgsManager
      */
     std::optional<const Command> GetCommand() const;
 
+    /**
+     * Get a normalized path from a specified pathlike argument
+     *
+     * It is guaranteed that the returned path has no trailing slashes.
+     *
+     * @param pathlike_arg Pathlike argument to get a path from (e.g., "-datadir", "-blocksdir" or "-walletdir")
+     * @return Normalized path which is get from a specified pathlike argument
+     */
+    fs::path GetNormalPath(std::string pathlike_arg) const;
```

Comment:
In commit "util: Add ArgsManager::GetNormalPath() function" (034f12a115e21b4205b532a3f946b129e83505d9)

Minor: Would vote for calling this `GetPathArg` to be consistent with `GetIntArg` `GetBoolArg`, instead of `GetNormalPath`

Reply:
Thanks! [Done](https://github.com/bitcoin/bitcoin/pull/24265#issuecomment-1030404670).